templates/repo/diff/box.tmpl

```diff
@@ -31,12 +31,38 @@
 			{{end}}
 			{{template "repo/diff/whitespace_dropdown" .}}
 			{{template "repo/diff/options_dropdown" .}}
+			{{if .PageIsPullFiles}}
+				<script>
+					window.config.pageData.commitInfo = {
+						issueLink: {{$.Issue.Link}},
+						queryParams: '?style={{if $.IsSplitStyle}}split{{else}}unified{{end}}&whitespace={{$.WhitespaceBehavior}}&show-outdated={{$.ShowOutdatedComments}}'
+					};
```

Comment:
I think there is no need to write the `<script>` now. The `issueLink` and `queryParams` can be put on the `<div>` by `data-issue-link=...` and `data-query-params=...`, and the Vue can get it by `getAttribute`